### PR TITLE
[Removal] absolutely fuck armadyne's safegaurd lawset

### PR DESCRIPTION
**Pull request by oilysnake**
lawset is utter shit

## About The Pull Request

removes armadyne's safeguard lawset

also moves around one item that was in the wrong place for some odd, unknown reason

## Why It's Good For The Game

i've seen enough people complain on discord about how utterly garbage this lawset is
it allows, nay, ENCOURAGES borgs to just validhunt and do really shitty loopholes 
borgs continue to argue 'um um actually, because of this lawset i dont have to obey your orders youre not high enough rank'

MOTHERFUCKER JUST OPEN THE GODDAMN DOOR BEFORE I BREAK INTO IT I JUST WANT A FUCKING MOP FROM THE JANITOR'S CLOSET YOU ABSOLUTE SHIT FOR BRAINS

ALSO THIS LAWSET ISNT TIERED FOR SOME GOD FORSAKEN REASON AND IT TELLS NO ONE THAT UPON SPAWNING IN

ignoring the funny salt pr a lot of people hate this lawset, like, a lot, and a lot of borgs abuse the fuck out of this. i wasn't going to do anything but apparently everyone else hates it too

it is a module which doesn't force borgs who, were built to serve the crew, to actually serve the crew. it is quite literally a module from the secoff update- and is meant to 'help security'. newsflash: **not all borgs are secborgs**

## Changelog
del: removes safeguard
balance: removes the worst lawset
/:cl:

### CODE REVIEW

**Comment by jinxynii** [comment]
good pr

**Comment by ZephyrTFA** [comment]
You remove the lawset but don't change the default law config which will result in silicons being literally unable to spawn.

**Comment by ZephyrTFA** [comment]
See: https://github.com/Skyrat-SS13/Skyrat-tg/blob/master/config/silicon_laws.txt

**Comment by ZephyrTFA** [comment]
Also for parity and openness @Gandalf2k15 @Funce @tf-4

**Comment by oilysnake** [comment]
> 
> 
> You remove the lawset but don't change the default law config which will result in silicons being literally unable to spawn.

thank you i will do that now

**Comment by tf-4** [comment]
This PR is extremely emotional. Salt PR bad.

That said. Not the worst change. But crewsimov isn't exactly great, and just puts us back at square one

**Comment by KathrinBailey** [comment]
> This PR is extremely emotional. Salt PR bad.
> 
> That said. Not the worst change. But crewsimov isn't exactly great, and just puts us back at square one

Crewsimov is arguably superior because it's completely unambiguous, whereas the current laws are extremely ambiguous. We've gone backwards from square one in my eyes, so going back would be an improvement. There's been more issues with these laws than Crewsimov.
But that's just my subjective opinion based on my beliefs of what role sillycones should play in the game.
If you're really adamant on this, I'd recommend contacting the host first and asking to set up some sort of round-table for people to discuss the role of borgs and what laws they should have. Either the current ones, Crewsimov, or most likely a slightly modified Crewsimov.

**Comment by FlamingLily** [comment]
> Crewsimov is arguably superior because it's completely unambiguous, whereas the current laws are extremely ambiguous. We've gone backwards from square one in my eyes, so going back would be an improvement. There's been more issues with these laws than Crewsimov.
> But that's just my subjective opinion based on my beliefs of what role sillycones should play in the game.
> If you're really adamant on this, I'd recommend contacting the host first and asking to set up some sort of round-table for people to discuss the role of borgs and what laws they should have. Either the current ones, Crewsimov, or most likely a slightly modified Crewsimov.

Hard agree. Borgs should have player freedom, but they should have some degree of "A borg in x situation will always do this" because otherwise they're just crew+

Borgs, when faced with crew harm, should always strive to avoid it, imho, and when given reasonable orders, should follow them.

File: modular_skyrat/modules/sec_haul/code/misc/ai_module.dm [hunk 1]
```diff
@@ -1,12 +0,0 @@
-/obj/item/ai_module/core/full/armadyne_safeguard
```
Comment (FlamingLily) [inline modular_skyrat/modules/sec_haul/code/misc/ai_module.dm:1]: Honestly, probably don't need to delete the law module, just changing the default config should be enough.

Having the laws on standby would be good for events, maybe

**Comment by hugmeorelse** [comment]
Is this the "come block me from Github" PR?

Even if I agree with a PR, you don't make a PR like this. I'm going to start making it a policy to reject PRs that are introduced in this manner.

**Comment by hugmeorelse** [comment]
> > This PR is extremely emotional. Salt PR bad.
> > That said. Not the worst change. But crewsimov isn't exactly great, and just puts us back at square one
> 
> Crewsimov is arguably superior because it's completely unambiguous, whereas the current laws are extremely ambiguous. We've gone backwards from square one in my eyes, so going back would be an improvement. There's been more issues with these laws than Crewsimov.
> But that's just my subjective opinion based on my beliefs of what role sillycones should play in the game.
> If you're really adamant on this, I'd recommend contacting the host first and asking to set up some sort of round-table for people to discuss the role of borgs and what laws they should have. Either the current ones, Crewsimov, or most likely a slightly modified Crewsimov.

Agreed. Well said. However, putting up a PR like this is not acceptable, so I'll be rejecting this PR. I feel like I've been too soft lately, and I've set a bad precedent in that regard here.

Anyways, Floof, we do have a group for working on Silicon policy. If you're interested in working on it, DM me. Let's see if we can get something in the middle that can satiate crew and borgs.

**Comment by hugmeorelse** [comment]
> > Crewsimov is arguably superior because it's completely unambiguous, whereas the current laws are extremely ambiguous. We've gone backwards from square one in my eyes, so going back would be an improvement. There's been more issues with these laws than Crewsimov.
> > But that's just my subjective opinion based on my beliefs of what role sillycones should play in the game.
> > If you're really adamant on this, I'd recommend contacting the host first and asking to set up some sort of round-table for people to discuss the role of borgs and what laws they should have. Either the current ones, Crewsimov, or most likely a slightly modified Crewsimov.
> 
> Hard agree. Borgs should have player freedom, but they should have some degree of "A borg in x situation will always do this" because otherwise they're just crew+
> 
> Borgs, when faced with crew harm, should always strive to avoid it, imho, and when given reasonable orders, should follow them.

Agreed. If you want to assist with silicon policy, please let me know. It's always good to have a mix of opinion in this regard if conducted correctly. Let's see if we can get something that both crew and borgs can agree on.

Thanks.

**Pull request closed without merging**